web/crux/templates/vaultwarden.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "Vaultwarden",
+  "description": "Alternative implementation of the Bitwarden server API written in Rust and compatible with upstream Bitwarden clients, perfect for self-hosted deployment where running the official resource-heavy service might not be ideal.",
+  "registries": [
+    {
+      "name": "Vaultwarden / Docker HUB",
+      "type": "hub",
+      "hub": {
+        "imageNamePrefix": "vaultwarden"
+      }
+    }
+  ],
+  "images": [
+    {
+      "name": "vaultwarden-server",
+      "registryName": "Vaultwarden / Docker HUB",
+      "image": "server",
+      "tag": "latest",
+      "config": {
+        "name": "vaultwarden",
+        "ports": [
+          {
+            "external": 80,
+            "internal": 80
+          }
+        ],
+        "volumes": [
+          {
+            "name": "data",
+            "path": "/data"
+          }
+        ],
+        "environment": [],
```

Comment:
We need `DOMAIN` environment variable, containing the site URL: https://vault.example.com/.